### PR TITLE
dockers for now - with variable definitions outside, vision + edge

### PR DESCRIPTION
**Pull request by chayim**
Changes to build edge, from edge - in an edge only fashion. Now, nightly we'll build **edge** dockers (specifically) twice daily.

We're doing this here, because dokcerhub webhooks can't (currently) submit to CircleCI, due to a need to process POST bodies. The same is true of GitHub actions - dockerhub would need to add a post body, which it cannot.

### CODE REVIEW

File: edge/Dockerfile [hunk 1]
```diff
@@ -11,32 +11,39 @@ ARG ARCH=x64
 ARG REDISAI_VERSION=1.0.2
 ARG REDISTIMESERIES_VERSION=1.4.6
 ARG REDISGEARS_VERSION=1.0.2
-ARG REDIS_VER=6.0.9
+ARG REDIS_VERSION=6.0.9
```
Comment (rafie) [inline edge/Dockerfile]: `docker.native` uses `REDIS_VER`.

Comment (chayim) [reply edge/Dockerfile]: Gah - I changed it to REDIS_VERSION throughout because it was prettier. *sighs and changed*

**Comment by rafie** [comment]
I'll approve but let's make sure we're tested.
Also: why do we need the Dockerhub build?

**Review by rafie — APPROVED** [review]

**Comment by chayim** [comment]
> I'll approve but let's make sure we're tested.
> Also: why do we need the Dockerhub build?

Testing wise, I've made sure we don't touch the default. As for the dockerhub build, it's phase one of a RedisAI use case.

**Pull request merged**